automate.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import print_function
+import subprocess
+
+
+def automate(ver):
+    comment = '"Added class names for Youtube versions ' + str(ver) + '"'
+    try:
+        subprocess.call("git pull && git add .", shell=True)
+        subprocess.call(["git commit -m {0}".format(comment)], shell=True)
+
+    except Exception as e:
+        print("Error occured")
+        print(e)
+
+    finally:
```

Comment:
If error occurs in executing git pull or git commit, then there's no point in pushing and creating a pull request.

Instead move the content of finally block into the try block itself.